.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Related Issue
```

Comment:
I believe the filename should be capitalized, i.e. `PULL_REQUEST_TEMPLATE.md`.

At least the one in the main opencue repo is: https://raw.githubusercontent.com/AcademySoftwareFoundation/OpenCue/master/.github/PULL_REQUEST_TEMPLATE.md

Also, feel free to copy-paste any of the language from that other PR template. For example the text on adding an issue number. It's good to keep things consistent between repositories when we can.

Reply:
Okk I will update it.